src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -211,6 +236,260 @@ public function deleteAction(Request $request, int $productId): Response
+     *     redirectQueryParamsToKeep={"id_category"},
+     *     message="You do not have permission to edit this."
+     * )
+     * @DemoRestricted(
+     *     redirectRoute="admin_products_v2_index",
+     *     redirectQueryParamsToKeep={"id_category"}
+     * )
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function updatePositionAction(Request $request): RedirectResponse
+    {
+        try {
+            $this->getCommandBus()->handle(
+                new UpdateProductPositionCommand(
+                    $request->request->get('positions'),
+                    $request->query->getInt('id_category')
+                )
+            );
+        } catch (CannotUpdateProductPositionException $e) {
+            $errors = $e->getErrors();
+            $this->flashErrors($errors);
+
+            return $this->redirectToRoute('admin_products_v2_index');
+        }
+        $this->addFlash('success', $this->trans('Successful update.', 'Admin.Notifications.Success'));
+
+        return $this->redirectToRoute('admin_products_v2_index');
+    }
+
+    /**
+     * Delete products in bulk action.
+     *
+     * @AdminSecurity(
+     *     "is_granted('delete', request.get('_legacy_controller'))",
+     *     redirectRoute="admin_products_v2_index",
+     *     message="You do not have permission to delete this."
+     * )
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function bulkDeleteAction(Request $request): RedirectResponse
+    {
+        try {
+            $this->getCommandBus()->handle(new BulkDeleteProductCommand(
+                    $this->getProductIdsFromRequest($request))
+            );
+            $this->addFlash(
+                'success',
+                $this->trans('Successful deletion.', 'Admin.Notifications.Success')
+            );
+        } catch (Exception $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
+        }
+
+        return $this->redirectToRoute('admin_products_v2_index');
+    }
+
+    /**
+     * Enable products in bulk action.
+     *
+     * @AdminSecurity(
+     *     "is_granted('update', request.get('_legacy_controller'))",
+     *     redirectRoute="admin_products_v2_index",
+     *     message="You do not have permission to edit this."
+     * )
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function bulkEnableAction(Request $request): RedirectResponse
+    {
+        try {
+            $this->getCommandBus()->handle(
+                new BulkToggleProductCommand(
+                    $this->getProductIdsFromRequest($request),
+                    true
+                )
+            );
+            $this->addFlash(
+                'success',
+                $this->trans('Successful enable.', 'Admin.Notifications.Success')
```

Comment:
I'm not sure to understand what you mean here. Could you give more details about this wording?